pkg/util/kernel/platform.go

```diff
@@ -8,7 +8,7 @@
 package kernel
 
 import (
-	gopsutilhost "github.com/shirou/gopsutil/v3/host"
```

Comment:
going back to our completely unmaintained fork feels like a mistake, what are we losing to fix the issue you found ? I would like to see an upstream PR fixing the issue so that we can go back to upstream ASAP

Reply:
Opened an upstream PR to fix this particular issue: https://github.com/shirou/gopsutil/pull/1584

Reply:
We will need this change too, which might be more controversial: https://github.com/DataDog/gopsutil/pull/44/files

Reply:
https://github.com/shirou/gopsutil/pull/1585